src/current/v23.1/cpu-profiler.md

```diff
@@ -0,0 +1,71 @@
+---
+title: CPU Profiler
```

Comment:
Suggestion: let's call this what it is - it's the "Automatic CPU Profiler". 

I think calling it "CPU Profiler" makes it sound more generic about CPU profiles as a whole. We should probably have separate docs describing CPU Profiles, but that's outside the scope of this page. That's why I think we should add the `Automatic` prefix in all the places we reference "CPU Profiler". WDYT?

Reply:
Changed all references to this feature to "Automatic CPU Profiler"